citadel/indico_citadel/models/search_id_map.py

```diff
@@ -177,6 +178,11 @@ class CitadelSearchAppIdMap(db.Model):
         )
     )
 
+    file = db.Column(
+        db.Integer,
+        nullable=True,
+    )
```

Comment:
Let's put that to the other columns (ie before all the relationships); and maybe rename it to something clearer or even change it to a bool...

Right now this seems to be the Attachment.id; but that's already in the `attachment_id` (which has a proper FK and relationship). So a bool to track whether the file has been uploaded should be enough...

Reply:
I had a bool before, but I thought it would be a better idea to keep a track of the AttachmentFile.id (as opposed to attachment id).

Reply:
Hm, then maybe store it as `attachment_file_id`, but with a FK - and I think currently your code sets it to `entry.id` which I thought was the attachment id, not the file id..

Reply:
Yes, entry is right now an `AttachmentFile`